tests/badge.spec.ts

```diff
@@ -48,7 +48,7 @@ test('Badge E2E Tests', async ({ page }) => {
 
   // Green Badge Test
 
-  const greenBadge = page.getByText("Go")
+  const greenBadge = page.getByText("Go", { exact: true })
```

Comment:
had an issue where the text "Go" was being picked up in other sample text for collapse tests. this is a quick fix, but probably not sustainable if other components will also use the same language. wondering if for the future, test files should be split up into 1 page per component, rather than one big file for all PRIME tests?

Reply:
We probably should.